extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -177,104 +196,153 @@ export class AmpIframe extends AMP.BaseElement {
 
 
   /** @override */
-  layoutCallback() {
-    if (!this.isClickToPlay_) {
-      this.assertPosition();
+  isRelayoutNeeded() {
```

Comment:
This is not needed (and actually rather dangerous). Currently, if `documentInactiveCallback` returns true, as it does, the state is reset to "needs layout" and `layoutCallback` is called. I know it's a bit cryptic and our next task :)